### PR TITLE
Updated the Photon OS Document Pages

### PR DESCRIPTION
**Pull request by dsoumay**
As discussed and reviewed by Hari, I have added new fields to the Permitted JSON field list in the page.

@Michael Roy, please merge this change if everything looks fine. Thanks.

### CODE REVIEW

**Comment by dsoumay** [comment]
Hi Michael,
I have included some more commits to this PR. Wanted some feedback on the same from the developers.
Can you please hold the merge till the other commits are also approved.

Thanks,
Soumay

**Review by ssahani — APPROVED** [review]

**Review by ssahani — APPROVED** [review]

**Review by ssahani — APPROVED** [review]

**Review by ssahani — APPROVED** [review]

**Comment by ssahani** [comment]
LGTM. thanks !

File: content/en/docs/administration-guide/pmd-nextgen/photon-mgmtd-web-REST-API/firewall-nftable-management.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	pmctl network delete-nft-table name <TABLE> family <FAMILY>
+
+Example:
+
+	>pmctl network delete-nft-table name test99 family inet
+
+## Show nftable ##
+
+To show an nftable, use the following command in `pmctl`:
+
```
Comment (ssahani) [inline content/en/docs/administration-guide/pmd-nextgen/photon-mgmtd-web-REST-API/firewall-nftable-management.md:32]: an -> a  Grammer

File: content/en/docs/administration-guide/pmd-nextgen/photon-mgmtd-web-REST-API/firewall-nftable-management.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+## Show all nftables ##
+
+To show all the nftables, use the following command in `pmctl`:
+
+	>pmctl network show-nft-table
+
+## Add nftable chain ##
+
+To add an nftable chain, use the following command in `pmctl`:
```
Comment (ssahani) [inline content/en/docs/administration-guide/pmd-nextgen/photon-mgmtd-web-REST-API/firewall-nftable-management.md:47]: ditto

**Pull request merged**